### ISSUE TITLE
GPA comments

### ISSUE THREAD

**Issue by mghasemi19** [mghasemi19/ML_EconEduc#4]
Some more comments need to be fixed in GPA paper:

Meisam's cooments:
- [ ] reference for all papers 
- [x] more investigation on data frame correlations for discrete variables
- [x] normal distribution for GPA and KS test in secction 3
- [x] search for regression metrics in other papers
- [ ] HP and overfit and loss fucntion description for NN
- [x] compare NN vs NN optimized in terms of having low metrics
- [ ] more text on BDT plot (pred minues observed)

Ahmad's comments:

- [x] Table 1 correction (drop t eh column and add it to the explainatation)
- [x] update table 1
- [x] search for regression metrics in other papers
- [x] study other papers mainly focus on metrics, models, overfit, hyperparameter optimization

**Comment by mghasemi19** [comment]
- [x] Add simple NN structure
- [ ] Regression overfit
- [x] Add R square for all models
- [x] Ask about acceptable values for R^2 and RMSE
- [ ] Ask about type model validity
- [x] 2 papers with regression models
- [x] Convert GPA regression to classification models
- [ ] Do more data engineering to select better data
